.github/scripts/cherryPick.sh

```diff
@@ -0,0 +1,20 @@
+echo "Attempting to cherry-pick $1"
```

Comment:
Are we forgetting the shebang up here?

Reply:
Yes! Odd that it was running locally for me, but we'll definitely want the shebang.